packages/mermaid/src/diagrams/class/classDb.ts

```diff
@@ -456,6 +457,20 @@ export const addClassesToNamespace = function (id: string, classNames: string[])
   }
 };
 
+export const setCssStyle = function (id: string, styles: string[]) {
+  const thisClass = classes[id];
+  if (!styles || !thisClass) {
+    return;
+  }
+  for (const s of styles) {
+    if (s.includes(',')) {
+      thisClass.styles = [...thisClass.styles, ...s.split(',')];
+    } else {
+      thisClass.styles?.push(s);
```

Comment:
```suggestion
      thisClass.styles.push(s);
```

If `styles` is no longer optional, I think we can remove the `?` field.